lib/segment/src/index/field_index/null_index/mmap_null_index.rs

```diff
@@ -202,7 +209,7 @@ impl MmapNullIndex {
 
 impl PayloadFieldIndex for MmapNullIndex {
     fn count_indexed_points(&self) -> usize {
-        self.has_values_slice.len()
+        self.has_values_slice.count_flags()
```

Comment:
If there a strong reason to change this?

This function is called on INFO, so it might be too expensive to run it every time

Reply:
Also, we aggregate indexed count for each index and choose the minimal one. So this will likely cause the index to be displayed as 0 in almost all cases

Reply:
Reverted 👍